hw2/03-count.html

```diff
@@ -1,34 +1,38 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>03 - Count</title>
-    <link
-      rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
-    <script defer src="03-count.js"></script>
-  </head>
-  <body>
-    <main>
-      <h1>Exercise 03 - Count</h1>
-      <hr />
-      <div>
-        <label for="userInput">Highlight all occurences of a word:</label>
-        <input type="text" id="userInput" />
-      </div>
-      <hr />
-      <div>
-        It was the best of times, it was the worst of times, it was the age of
-        wisdom, it was the age of foolishness, it was the epoch of belief, it
-        was the epoch of incredulity, it was the season of Light, it was the
-        season of Darkness, it was the spring of hope, it was the winter of
-        despair, we had everything before us, we had nothing before us, we were
-        all going direct to Heaven, we were all going direct the other way—in
-        short, the period was so far like the present period, that some of its
-        noisiest authorities insisted on its being received, for good or for
-        evil, in the superlative degree of comparison only.
-      </div>
-    </main>
-  </body>
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>03 - Count</title>
+        <link
+            rel="stylesheet"
+            href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+        />
+        <style>
+            mark {
+                background-color: yellow; /* Change this to the desired highlight color */
+            }
+        </style>
+    </head>
+    <body
+        class="flex justify-content-center align-content-center row"
+        style="background-color: rgb(0, 66, 92)"
+    >
+        <!-- The blue does not exist within the bootstrap classnames -->
+        <main class="col-6 bg-light container mt-5 rounded">
+            <div class="">
+                <h1 class="p-3 text-center fw-bold">Exercise 03 - Count</h1>
+                <hr class="" />
+                <div>
+                    <label for="userInput"
+                        >Highlight all occurrences of a word:</label
+                    >
+                    <input type="text" id="userInput" />
+                </div>
+                <hr />
+                <div id="toSearch"></div>
+            </div>
+            <script defer src="03-count.js"></script>
```

Comment:
Interesting choice to move the text to the js file. I think that is a smart move regarding ease of use.